<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>423</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Downloader</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="YouTubeDownloader">
    <attribute name="title">
     <string>YouTube Downloader</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_YT_URL">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Please Enter or Paste YouTube URL Here</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_YT_SAVE">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Browse and Select Your Desired Location to Save the Video File</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select Video Quality</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_YT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>481</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="push_YT_DL">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>320</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QComboBox" name="YT_Video_Quality">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select Audio Quality</string>
     </property>
    </widget>
    <widget class="QComboBox" name="YT_Audio_Quality">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>341</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Download Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="YT_Download_Type">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Combine Audio &amp; Video</string>
     </property>
    </widget>
    <widget class="QComboBox" name="YT_IsCombine">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>220</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="push_YT_Browse">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="FileDownloader">
    <attribute name="title">
     <string>File Downloader</string>
    </attribute>
    <widget class="QPushButton" name="push_FD_DL">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_FD_SAVE">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Browse and Select Save Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_FD_URL">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Please Enter or Paste URL Here</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_FD">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>481</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="push_FD_Browse">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_FD">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>471</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
